pkg/codegen/internal/test/testdata/aws-eks.pp.go

```diff
@@ -71,7 +71,7 @@ func main() {
 			vpcSubnet = append(vpcSubnet, __res)
 		}
 		var rta []*ec2.RouteTableAssociation
-		for key0, _ := range zones.Names {
+		for key0 := range zones.Names {
```

Comment:
Did you happen to run an auto-formatter or linter on these from your IDE? These changes to the go test files look accidental.

Reply:
I did in response to a lint failure. Reverted.